chain/gen/genesis/miners.go

```diff
@@ -143,7 +143,7 @@ func SetupStorageMiners(ctx context.Context, cs *store.ChainStore, sroot cid.Cid
 				return cid.Undef, err
 			}
 			params := &power.EnrollCronEventParams{
-				EventEpoch: miner.ProvingPeriod,
+				EventEpoch: miner.ProvingPeriod + power.WindowedPostChallengeDuration,
```

Comment:
For correctness this should of probably been `st.PoStState.ProvingPeriodStart` to start with. Not an issue until someone resolves the TODO above about randomizing genesis miners.